frontend/src/scenes/PreflightCheck/logic.ts

```diff
@@ -36,11 +36,11 @@ export const preflightLogic = kea<preflightLogicType<PreflightStatus, PreflightM
         ],
         realm: [
             (s) => [s.preflight],
-            (preflight): 'cloud' | 'hosted' | null => {
+            (preflight): 'cloud' | 'hosted' | 'hosted-clickhouse' | null => {
```

Comment:
Maybe it's better to get rid of the selector altogether and use the value inside the `preflight` object instead, avoid duplicate information.